.github/workflows/crawler.yml

```diff
@@ -0,0 +1,76 @@
+name: crawler
+
+on:
+  workflow_dispatch:    
+  push:
+    branches: [ "main" ]
+
+jobs:
+  install-zcashd:
+    runs-on: ubuntu-latest
+    steps:
+      - name: clone zcashd
+        run: git clone https://github.com/zcash/zcash
+      - name: build zcashd
+        run: |
+          cd zcash
+          ./zcutil/build.sh -j$(nproc)
+      - uses: actions/upload-artifact@v2
+        with:
+          name: zcashd-fetch-params
+          path: ./zcash/zcutil/fetch-params.sh
```

Comment:
Do we do this twice on purpose? the installation steps?

Reply:
I couldn't get the node up unless we fetch the params first. So, to avoid uploading the whole `zcash` directory, I just upload the two things we need.